be/src/exec/pipeline/pipeline_builder.cpp

```diff
@@ -22,11 +22,15 @@ void PipelineBuilderContext::maybe_interpolate_local_exchange(OpFactories& pred_
         pred_operators.emplace_back(std::move(local_exchange_sink));
         // predecessor pipeline comes to end.
         add_pipeline(pred_operators);
+
+        OpFactories operators_source_with_local_exchange;
```

Comment:
no need to modify this function,  since pred_operators is moved when local exchange is interpolated, at this moment, pred_operator is clear and can be used.